Sources/ProjectSpec/Dependency.swift

```diff
@@ -31,6 +33,17 @@ public struct Dependency: Equatable {
         self.link = link
         self.implicit = implicit
         self.weakLink = weakLink
+        self.platform = platform
+    }
+    
+    public enum Platform: Equatable {
+        case all
+        case iOS
+        case mac
```

Comment:
Should we use `macOS` to match the iOS and the platform names from the rest of Xcodegen

Reply:
Updated in dfc471d8511f37738bfb9a84915d822bcc299f1e.